tpm2tools/pcr.go

```diff
@@ -67,6 +68,77 @@ func ReadPCRs(rw io.ReadWriter, sel tpm2.PCRSelection) (*proto.Pcrs, error) {
 	return &pl, nil
 }
 
+// SealCurrent represent current PCRs states.
```

Comment:
Here and below, we need to rephrase these comments.

Idea:
```go
// SealCurrent seals data to the specified PCR selection.
// SealTarget predicatively seals data to the specified PCR values.
// SealingOpt specifies the PCR values that should be used for Seal().

// CertifyCurrent certifies that a selection of PCRs have the same value they did when sealing the data.
// CertifyExpected certifies that the TPM had a specific set of PCR values when sealing the data.
// CertifyOpt determines which sets of PCR values pass certification in Unseal().
```

Reply:
// SealCurrent seals data to the current specified PCR selection.
// SealTarget predicatively seals data to the given specified PCR values.
// SealingOpt specifies the PCR values that should be used for Seal().

// CertifyCurrent certifies that a selection of current PCRs have the same value when sealing.
// CertifyExpected certifies that the TPM had a specific set of PCR values when sealing.
// CertificationOpt determines if the given PCR value can pass certification in Unseal().